doc/man/man3/crypto_lock.3monocypher

```diff
@@ -149,8 +149,10 @@ and
 .It Fa ad_size
 length of the additional data.
 .Sy That length is not authenticated.
-If it is transmitted over the wire, it should be appended to the
-additional data, and authenticated with it.
+If the additional data is of variable length, the length should be appended to
+.Fa ad
+so it gets authenticated, and should be extracted from the end of the
```

Comment:
Maybe "so that"? It feels like that would flow better, but YMMV.

Reply:
My mileage would keep "that" out. :smile: